packages/storage-plugin/src/storage.plugin.ts

```diff
@@ -9,37 +9,40 @@ import {
   actionMatcher,
   NgxsNextPluginFn
 } from '@ngxs/store';
+import { tap } from 'rxjs/operators';
 
 import {
+  StorageEngine,
   NgxsStoragePluginOptions,
-  NGXS_STORAGE_PLUGIN_OPTIONS,
   STORAGE_ENGINE,
-  StorageEngine
+  NGXS_STORAGE_PLUGIN_OPTIONS
```

Comment:
Have we lost our ordering of the imports here? Max's IDE will want to fix it ;-)